tests/unit/httpServer/stats/model/statsManager.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { StatsManager } from '../../../../../src/httpServer/stats/models/statsManager';
+import { QueueProvider } from '../../../../../src/queue/queueProvider';
+
+describe('statsManager', () => {
+  describe('#getStats', () => {
```

Comment:
add a test case requiring the resolved value to be a promise

Reply:
`resolves` already tests for it